### PR TITLE
feat(srcPaths): Support custom source paths other than 'src'

### PR DESCRIPTION
**Pull request by markdalgleish**
## Description

By default, sku expects your source code to be in a directory named `src` in the root of your project. If your source code needs to be arranged differently, you can provide a `srcPaths` array.

## Example Usage

 ```js
module.exports = {
  ...,
  srcPaths: [
    'src',
    'docs/src'
  ]
};
```

### CODE REVIEW

File: scripts/format.js [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
 
 const args = require('../config/args').argv;
 
 console.log(chalk.cyan('Formatting source code with Prettier'));
 
-const filePattern = args.length === 0 ? defaultPath : args;
+const filePattern =
+  args.length === 0
+    ? builds[0].paths.src.map(
+        srcPath => `${path.relative(process.cwd(), srcPath)}/**/*.js`
```
Comment (mbfisher) [inline scripts/format.js]: Do we want to limit prettier to just JS files?

File: test/test-cases/custom-src-paths/custom-src-paths.test.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+const { promisify } = require('es6-promisify');
```
Comment (mbfisher) [inline test/test-cases/custom-src-paths/custom-src-paths.test.js]: 👏👊

File: scripts/format.js (hunk 1 above)
Comment (markdalgleish) [reply scripts/format.js]: Hmmmm, good point. We weren't limiting this previously. This wasn't a conscious decision.

Comment (markdalgleish) [reply scripts/format.js]: @mbfisher Fixed! 😄

**Pull request merged**